tests/test_trello_bookmarks_qa.py

```diff
@@ -306,7 +306,7 @@ def test_run(self):
                 self.assertGreater(record_count_2, 0)
 
                 # Assert that we are capturing the expected number of records for inc streams
-                self.assertEqual(record_count_1, len(expected_records_1.get(stream, [])),
+                self.assertGreater(record_count_1, len(expected_records_1.get(stream, [])),
```

Comment:
I think this should be assertGreaterEqual(). If this is passing for all streams that is a concerning. That means our expectations are always off.

Reply:
Nevermind I forgot we only have 1 inc stream here.